.github/workflows/action_tests.yaml

```diff
@@ -145,3 +145,46 @@ jobs:
           cd local-action
           npm install
           ./action_tests/assert.js all-hold-the-line-no-upload-id
+
+  payload:
+    name: ${{ matrix.description }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - description: pull-request-payload
+            payload_path: action_tests/pull_request_test_payload.json
+          - description: trunk-merge-payload
+            payload_path: action_tests/trunk_merge_test_payload.json
+          - description: all-payload
+            payload_path: action_tests/all_test_payload.json
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: local-action
```

Comment:
we should be able to use the src-repo / repo-under-test pattern in the other tests to bootstrap the test repo for these